### PR TITLE
azion-cli 1.5.1 (new formula)

### PR DESCRIPTION
**Pull request by patrick-menoti-azion**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Review by p-linnane — CHANGES_REQUESTED** [review]
Everything in homebrew-core must build from source. We are not able to accept pre-built binaries like this.

**Comment by PatrickMenoti** [comment]
> Everything in homebrew-core must build from source. We are not able to accept pre-built binaries like this.

Working on it!

**Comment by p-linnane** [comment]
I've rewritten the formula to use a more standard `go` build process.

**Comment by PatrickMenoti** [comment]
> I've rewritten the formula to use a more standard `go` build process.

May I edit your re-write to add the other LDFLAGS we use in this project?

**Comment by p-linnane** [comment]
> May I edit your re-write to add the other LDFLAGS we use in this project?

Go for it. Please just squash the commits into one with the same message that I used.

**Comment by p-linnane** [comment]
Looks like this does not actually meet our notability criteria:
```
Full audit azion-cli --online --new-formula output
  azion-cli
    * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

**Comment by marcusgrando** [comment]
👍 💯

**Comment by LucasGeneroso** [comment]
+ 1

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Comment by p-linnane** [comment]
We can waive the notability check since it seems there are a good number of people who want this included.

**Comment by PatrickMenoti** [comment]
> We can waive the notability check since it seems there are a good number of people who want this included.

I noticed that you changed the name from azion to azion-cli. Why is that? 
I was hoping to be able to install it by running: brew install azion

**Comment by p-linnane** [comment]
> I noticed that you changed the name from azion to azion-cli. Why is that? I was hoping to be able to install it by running: brew install azion

To match the repo name, and distinguish it from whatever the platform/service it interacts with is.

File: Formula/a/azion-cli.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class AzionCli < Formula
```
Comment (chenrui333) [inline Formula/a/azion-cli.rb:1]: if you are not sure about this name, maybe you should ask first?

https://github.com/Homebrew/homebrew-core/pull/152353

**Pull request merged**